dev-util/kdstatemachineeditor/kdstatemachineeditor-9999.ebuild

```diff
@@ -21,21 +21,21 @@ IUSE="doc test"
 SLOT="0"
 
 RDEPEND="
-	$(add_qt_dep qtcore)
-	$(add_qt_dep qtdeclarative 'widgets')
-	$(add_qt_dep qtgui)
-	$(add_qt_dep qtnetwork)
-	$(add_qt_dep qtwidgets)
+	dev-qt/qtcore
```

Comment:
All your `dev-qt/` dependencies need to be fixed to `slot 5` (maybe I was not clear enough above).

Reply:
My bad.

Reply:
slots are not fixed yet?